server/message_hooks.go

```diff
@@ -416,8 +417,15 @@ func (p *Plugin) SendChat(srcUser string, usersIDs []string, post *model.Post) (
 			continue
 		}
 
+		fileNameData := strings.Split(fileInfo.Name, ".")
+		if len(fileNameData) < 1 {
+			continue
+		}
+
+		fileName, fileExtension := strings.Join(fileNameData[:len(fileNameData)-1], "."), fileNameData[len(fileNameData)-1]
```

Comment:
Use this - https://cs.opensource.google/go/go/+/go1.20.5:src/path/filepath/path.go;l=273